.travis.yml

```diff
@@ -12,8 +12,11 @@ matrix:
       env: TYPO3_VERSION="^8.7"
 
 env:
-  - TYPO3_VERSION="^7.6"
-  - TYPO3_VERSION="^8.7"
+  matrix:
+    - TYPO3_VERSION="^7.6" DEPENDENCIES=latest
```

Comment:
having an env var holding the additional --prefer-lowest argument directly would simplify the setup below a lot.

we could even add that directly to the require statement and would not need to have additional require command

Reply:
On it.

Reply:
Done and repushed.